tests/v4/test_particleset_execute.py

```diff
@@ -154,8 +154,9 @@ def test_uxstommelgyre_pset_execute(verbose_progress):
         runtime=np.timedelta64(10, "m"),
         dt=np.timedelta64(60, "s"),
         pyfunc=AdvectionEE,
-        verbose_progress=verbose_progress,
     )
+    assert utils.hash_float_array([p.lon for p in pset]) == 1165396086
```

Comment:
I wondered, it this sensitive to computer platform because we hash _all_ digits? We’re working with floats, after all. Why don’t we round it first, so that the hash becomes more like np.isclose?